.setup/schema.3.cql

```diff
@@ -56,6 +56,10 @@ create type payment (
   subtotal double, --what should have been paid (inc-tax)
   total double, --what should have been paid for the whole invoice (inc-tax)
   payment double, --payment = actual amount customer paid
+  currency text, --Ex EUR, USD
+  country text, --iso2 ex. US, DE, AU
+  rcode text, --State ex. CA, BW
+  region text, --State ex. California
```

Comment:
Are these missing in https://github.com/ichliebedich/cql/pull/343 ?
Should be created like the new `updated` columns below?

Reply:
I don't think so but we can look. I think it might be in another PR if not but I'll check.

Reply:
Yes I thought so, these changes were made 2 months ago in this commit https://github.com/ichliebedich/cql/pull/334/files

Reply:
@dioptre @IvanPetkow 
I have updated https://github.com/ichliebedich/cql/pull/343 and it's ready for merge.
We want to merge this here, too, but we haven't and we don't want to merge https://github.com/ichliebedich/cql/pull/334 right now. 
But mentioned fields are in there...
Please double check https://github.com/ichliebedich/cql/pull/334 for any dependency required for this task here, too and if necessary, please add them (e.g. fields listed above) to https://github.com/ichliebedich/cql/pull/343